### DISCUSSION TITLE
Matching 3rd party data sets to Overture

### DISCUSSION THREAD

**Discussion by ibnt1** [OvertureMaps/data#285] [Discussion]
For context see [here](https://docs.overturemaps.org/gers/) for an introduction to GERS.

A 3rd party data owner can make their data feed more interoperable by matching it with Overture and assigning GERS IDs to their features.
We have provided [here](https://github.com/OvertureMaps/schema/tree/dev/gers/examples/python) an example python script that matches a small GPS traces dataset with Overture road segments for demonstrative purposes. This is intended to present an exploratory starting place for data owners that don't already have a matching infrastructure setup.

Because the match scenarios are so broad and there are many approaches and tools already existing, we would like to discuss and learn from the community what are preferred methods, tools or libraries for such match scenarios. Please reply with your thoughts and ideas, thanks!

**Comment by tjwebb** [comment]
The link you provided (https://docs.overturemaps.org/gers/) is no longer working.

**Reply by ibnt1** [reply]
GERS introduction page was moved here: https://docs.overturemaps.org/gers/ thanks!
